packages/amplify-graphql-transformer-core/src/cdk-compat/stack-synthesizer.ts

```diff
@@ -12,18 +13,12 @@ export class TransformerStackSythesizer extends LegacyStackSynthesizer {
   private readonly mapingTemplateAssets: Map<string, string> = new Map();
   private _deploymentBucket?: CfnParameter;
   private _deploymentRootKey?: CfnParameter;
-  private boundStack?: Stack;
```

Comment:
StackSynthesizer introduced `boundStack` property and similar handling as this type.